frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +244,14 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    self:onChangeSize("decrease")
```

Comment:
It's better to show a Notification beforehand to let user know we are rerendering due to font size changes. Otherwise they might get the false impression that koreader got stuck for no reason.